components/suggestion/provider.tsx

```diff
@@ -1,7 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {ProviderResults} from 'components/forward_post_modal/forward_post_channel_select';
```

Comment:
it would be better to define the `ProviderResult` type here and import it in the `forward_post_modal.tsx` instead